src/test/editor-service/editor-service_test.ts

```diff
@@ -450,59 +461,6 @@ function editorTests(editorFactory: (basedir: string) => EditorService) {
                     }]);
         });
   });
-
-  suite('getWarnings', function() {
```

Comment:
Turns out they'd gotten duplicated by some merge^Wrebase along the way. See `suite('getWarningsForFile', ...)` above.